source/sync/go-to-production/optimize-sync-atlas-usage.txt

```diff
@@ -38,20 +42,25 @@ App Services stores history in your synced Atlas cluster.
 
 .. _backend-compaction:
 
+Optimization Methods
+--------------------
+
+There are two methods for optimizing utility data synced in you cluster
+depending on whether your app uses Flexible or Partition-Based Sync:
+
+1. Backend Compaction: Used for :ref:`Partition Based Sync <partition-based-sync>` apps
+#. Trimming: Used for :ref:`Flexible Sync <flexible-sync>` apps
```

Comment:
I would reverse the order of this list. We prefer Flexible Sync and should recommend it first.